config/routes.rb

```diff
@@ -10,11 +10,15 @@
 
   resources :outcomes do
     resources :assessments
-    resources :direct_assessments
+    resources :direct_assessments, shallow: true
     resources :indirect_assessments
     resources :surveys
     resources :participations
     resources :other_assessments
     resources :outcome_alignments
   end
+
+  resources :direct_assessments, only: [] do
```

Comment:
You're right, this probably isn't a good use case for shallow routes. Going to ditch the `shallow: true` but keep the idea.